### ISSUE TITLE
Specify which ML algorithms should be used

### ISSUE THREAD

**Issue by shahril96** [automl/auto-sklearn#602]
How can I specified out which ML algorithms should be used when pruning the hyper-parameters?

For example, if I want auto-sklearn to only prune SVM SVC and Gradient Boosting, but not the others eq; AdaBoost, etc.

I have been looking this elsewhere (official website) and this issues tracker, but seems can't find it.
